core/templates/dev/head/profile/profile.html

```diff
@@ -128,11 +128,10 @@ <h2 class="oppia-profile-username-small-screen">
       </md-content>
 
       <span class="oppia-profile-portfolio-pages" ng-if="numUserPortfolioExplorations > 6">
-        <li class="glyphicon glyphicon-arrow-left" ng-disabled="currentPageNumber == 0" ng-click="goToPreviousPage()"></li>
-        Showing <[currentPageNumber * PAGE_SIZE + 1]> - <[Math.min(numUserPortfolioExplorations, (currentPageNumber + 1) * PAGE_SIZE)]> of <[numUserPortfolioExplorations]>
-        <li class="glyphicon glyphicon-arrow-right" ng-disabled="(currentPageNumber + 1) * PAGE_SIZE >= userEditedExplorations.length"
-            ng-click="goToNextPage()">
-        </li>
+        <i class="material-icons" ng-disabled="currentPageNumber == 0"
```

Comment:
@souravbadami 
The first image of misplaced arrows that you gave (one arrow was above and one arrow was below the "Showing 6-7" text) was due to the <li> tag.
When I changed it to ``` <i> ``` the arrows and the text were aligned in the same line i.e the lastest image i uploaded was due to using ``` <i> ``` instead of ``` <li> ```